aws/resource_aws_ses_configuration_set_test.go

```diff
@@ -97,9 +73,34 @@ func testAccCheckAwsSESConfigurationSetExists(n string) resource.TestCheckFunc {
 	}
 }
 
-var escRandomInteger = acctest.RandInt()
-var testAccAWSSESConfigurationSetConfig = fmt.Sprintf(`
+func testAccCheckSESConfigurationSetDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*AWSClient).sesConn
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aws_ses_configuration_set" {
+			continue
+		}
+
+		_, err := conn.DescribeConfigurationSet(&ses.DescribeConfigurationSetInput{
+			ConfigurationSetName: aws.String(rs.Primary.ID),
+		})
+
+		if err == nil {
```

Comment:
small change: to validate the error @ L89, we'll want to check that the err object exists with `err != nil`